api/openapi/model-registry.yaml

```diff
@@ -71,7 +71,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/ArtifactCreate"
+              $ref: "#/components/schemas/Artifact"
```

Comment:
Will this renaming change the generated names for the downstream clients? Ie, in the python client would this change the data-types that the end-user needs to import from the package?

Reply:
This is addressed now!